features/strict_mode.feature

```diff
@@ -5,7 +5,6 @@ Scenario: StrictMode DiscWrite violation
     Then I should receive a request
     And the request is a valid for the error reporting API
     And the exception "errorClass" equals "android.os.StrictMode$StrictModeViolation"
-    And the exception "message" equals "policy=262145 violation=1"
```

Comment:
The policy value differs on the Travis CI emulator as it's a different API level, and all we really care about here is that the metadata value was set